PostBuild.sh

```diff
@@ -480,12 +482,28 @@ function authselectInit {
    err_exit "Succeeded initializing authselect" NONE
 }
 
+# Disable subscription-manager
+function DisableSubscriptionManager {
+  local YUM_CONF
+  YUM_CONF=$( readlink -f /etc/yum/pluginconf.d/subscription-manager.conf )
+  if [[ ${SUBSCRIPTION_MANAGER} == "enabled" ]]
+  then
+     return
+  fi
+
+  err_exit "Attempting to disable subscription-manager service... " NONE
+  chroot "${CHROOTMNT}" /bin/sed -i '/^enabled/s/1$/0/' "${YUM_CONF}" || \
```

Comment:
Do we need to gate this on existence of whatever package installs this file, or maybe just on the existence of the file?

Reply:
Yeah. For some reason I thought it was an RPM with a dependency-graph. However:

```bash
# rpm -q --whatrequires $( rpm --qf '%{name}\n' -qf $( readlink -f /etc/yum/pluginconf.d/subscription-manager.conf ) )
no package requires subscription-manager
# rpm -q --whatprovides $( rpm --qf '%{name}\n' -qf $( readlink -f /etc/yum/pluginconf.d/subscription-manager.conf ) )
subscription-manager-1.28.36-3.el8_8.x86_64
```

So, I'll add a file-existence check.

Reply:
Added the existence-check to the function's initial-exit criteria.